### DISCUSSION TITLE
How do I configure a third party service that supports apprise to use one of my stored configs?

### DISCUSSION THREAD

**Discussion by jdevera** [caronc/apprise-api#205] [Q&A] [answered]
I've setup apprise-api with my very useful configurations and have been using the `notify` endpoint happily.

Now I am trying to configure changedetection.io to use this but it does support apprise directly. My understanding from the docs is that I can get the specific configuration with the `/get/$config` endpoint and import that as the configuration for the apprise CLI. The problem I'm seeing is that the `/get/$config` endpoint needs a `POST`, not a `GET` to return the config.

Maybe I'm tackling this the work way, my question could be more generic:

Once I have some configurations in apprise-api, how can I use the `apprise` CLI from another box to send the notifications?

**Comment by caronc** [comment, marked as answer]
I would just use the [AppriseAPI plugin](https://github.com/caronc/apprise/wiki/Notify_apprise_api) (`apprise://`) built into Apprise.

So in your case:
- `apprise://your.apprise.server/token`

If you need to trigger a tag specifically that you've setup:
- `apprise://your.apprise.server/token?tags=devops`


If you password protected this:
- `apprise://user:pass@your.apprise.server/token?tags=devops`

**Comment by jdevera** [comment]
I just realised that the instructions were on the web UI itself:

```
# Use Apprise to retrieve your configuration:
apprise --body="test message" --config=https://apprise-api.url/get/KEY
```

And I think the reason why my experiments failed was because I use a custom CA for my https certificate. I already made it work with the usual REQUESTS_CA_BUNDLE/SSL_CERT_FILE trick.

However, now looking at the plugin suggestion, I think it's finer-grained and more like what I want to use. Thanks @caronc!
